private-templates-service/client/src/store/page/types.ts

```diff
@@ -1,15 +1,19 @@
 export interface PageState {
-  currentPageTitle: string;
-  currentPage: string;
+  currentPageTitle?: string;
+  currentPage?: string;
+  modalOpen?: string;
 }
 
 // Action Types
 export const NAVIGATION = "NAVIGATION";
+export const OPEN_MODAL = "OPEN_MODAL";
```

Comment:
nit: If this modal is the 'share modal' specifically, can we rename to this.

Reply:
This is for all modals, publish, unpublish and share.